blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/rest/BlueTrendsApiTest.java

```diff
@@ -37,8 +37,8 @@ public void getTrendsListFreestyle() throws IOException {
 
     @Test
     public void getJUnitTrends() throws Exception {
-        URL resource = Resources.getResource(getClass(), "BlueJUnitTestResultTest.jenkinsfile");
-        String jenkinsFile = Resources.toString(resource, Charsets.UTF_8);
+        URL resource = getClass().getResource( "BlueJUnitTestResultTest.jenkinsfile");
```

Comment:
What is up with these spaces? 
```suggestion
        URL resource = getClass().getResource("BlueJUnitTestResultTest.jenkinsfile");
```